src/components/checkbox/checkbox.ts

```diff
@@ -248,6 +215,35 @@ export class MdCheckbox implements ControlValueAccessor {
     }
   }
 
+  /*
+   * We use a hidden native input field to handle changes to focus state via keyboard navigation,
+   * with visual rendering done separately. The native element is kept in sync with the overall
+   * state of the component.
+   */
+  onInputFocus() {
+    this.hasFocus = true;
+  }
+
+  onInputBlur() {
+    this.hasFocus = false;
+  }
+
+  toggle() {
```

Comment:
Add a JsDoc for `toggle`
(I know it's obvious, but we're eventually going to pull some API docs from these)